### DISCUSSION TITLE
User could not be added

### DISCUSSION THREAD

**Discussion by Yggdrasills** [community/community#35582] [Repositories] [answered]
Hello. Please tell me why I can not add a user to a private repository? The user is displayed in the list of found users, but after an attempt is made to invite, the page is refreshed and an `User could not be added` error  appears

**Comment by fadyat** [comment]
Any screenshots?

Go to Settings > Collaborators and select a user.
They will receive an invitation and can accept or decline it.

It's always works fine for me.

**Reply by Yggdrasills** [reply]
Any new user can be added to the repository without problems. But there is one account that I can only invite to the public repository. If it is private, then an error is shown. Also, if you open a repository, add a user and then close it, the user does not see this repository

**Comment by m-hugo** [comment, marked as answer]
Github users from US-sanctioned regions cannot interact in any way with private repositories

edit: since this thread has blown up i just want to clarify that i'm not affiliated with github or claimed this is THE explaination, this is just a partial explaination to the "can't add a specific user if the repo is private scenario" if the user is not in https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls#on-which-countries-and-territories-are-us-government-sanctions-applied and you want gihub's help i recommend finding out the following:
- can both users create private repos
- is this specific to a single user you want to add, multiple users, or all of them
- is this specific to private repos or are public ones also affected

and then try contacting mailto:developer@githubsupport.com with all the info

**Reply by WangleLine** [reply]
what the fuck, why

**Reply by tonybirol** [reply]
Just had this weird issue today, there's no way that this is the reason and answer why I can't add a collab into a private repo. In case this is the answer, why?

**Reply by LEBEST0** [reply]
J'ai le même problème actu

**Reply by Vaserrrz** [reply]
how can this be fixed? I mean can GitHub revoke the sanction?

**Reply by mhamidjamil** [reply]
why is this shit mentioned as an answer it's not the answer everyone here is looking for just because of this uncomplete answer the support show us this shit, i'm trying to add my class fellow as a collaborator but i also get the same error but the key point is my account was deleted by the GitHub for more then 5 months and i got it back last week after the recovery (thanks to GitHub support not for replying my mails after 5 months but to recover this account) I'm unable to perform some actions but before my account was deleted i have added 3 collaborators (my class fellows on the same repo)

**Reply by mhamidjamil** [reply]
in my case, my account was flagged (suspended due to suspicious activity) and when it was recovered i was unable to add user then i contact support and they ask me to delete other 3 accounts and 2 was mine but not the third one, in short after resolving some dependencies my account was again start working as a normal account

**Reply by blackysky** [reply]
That totally solved my problem, thanks!
My Crimean friend is now helpless.

**Comment by randevb** [comment]
I am from sri lanka, but I am unable to add another user from sri lanka. why is that?
I checked moments ago, Sri Lanka is not sanctioned by US

**Comment by kclubdigital** [comment]
Hello support. I am based out of Indonesia but is a holder of Australian passport, I have a repository which I am inviting my collaborator also from Australia, and also other countries like Indonesia, from last week I am not able to add any collaborators to my repository on multiple accounts. Every time I am adding user either via the username or email, the result is "User could not be added" error is posted. Can you please let me know how I can resolve this ongoing issue.

**Comment by fundly-tech** [comment]
I am facing same issue. I am from India and collaborators invited by me are in India. Every time, I am getting same message "User can not be added". Why?

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.

**Comment by a7iiid** [comment]
I am facing the same issue. I am from Palestine, and collaborators invited by me are in Palestine. Every time, I am getting the same message: "User cannot be added.". Why?

**Comment by gamtroller1** [comment]
It's happening to me too.

**Comment by ChetanReddyC** [comment]
the same problem ive even tried as private and public repos but it isn't working still the same error , if any one got a solution for this situation iam requesting to share it   

-thanks in advance.

**Comment by SepidehDastoorani** [comment]
I have the same problem, and I can't add my professor to the project!!!

**Comment by abhishekraaut** [comment]
please give solution, After I created a Repository on Github and tried to invite collaborators, I am shown the message 'User could not be added'.

**Comment by SmartekIbero** [comment]
I have the same problem. Any idea how to solve it??

**Comment by lmolinario** [comment]
I have the same problem too.

**Comment by sekeiryuutei** [comment]
Hola, tengo el mismo problema, como pudieron resolverlo?

**Comment by krinya** [comment]
Having the same issue, it isn't very pleasant. Not able to work...

**Comment by aqdas-united** [comment]
Having the same issue, Please help me resolving this issue

**Comment by Hannes821** [comment]
having the same issue, Germany, Ireland, Slowakia involved in my team.
I dont think this is about sanctions!
cheers

**Comment by M-125** [comment]
same issue with a guy from germany

**Comment by Adatasol-Inc** [comment]
There doesn't appear to be any answers to this yet - the accepted answer does not seem to be valid for all situations, as many here have said.

I'm in the US trying to invite somebody else from the US to my Private Project (free tier Github).

**Comment by menguzat** [comment]
yeah github, wtf??

I'm from Turkey, my collab from turkey I can use github fine. so can she. but she can't add me as collaborator.

and why do you guys not answer? this is not a closed issue by far!

**Comment by ChetanReddyC** [comment]
Hey even i also had the same issue,but i ive sent a mail to this mail id 'developer@githubsupport.com' ,by describing all my issues with my account (can't be able to add a collaborator, can't authenctated with the third party services with github) , ive got a manual reply with in couple of conversations MY ALL ISSUES WAS RESOLVED , now every thing works fine! ,it took averagely 7days for me to settle this.

**Comment by Olliug91** [comment]
In my case, the problem was the type of account. I was working with an enterprise account, not a personal account. And GitHub answer me that is not possible collaborations with a non pay enterprise account. Solution was transform that enterprise account in a personal account. And it is working now.

**Comment by AkiraYuki-04** [comment]
If you're unable to add or collab with users on GitHub, it may be because your account is pending enrollment for two-factor authentication (2FA). This period is called the ### "2FA enrollment period."

During this time, your account may not function properly, including actions like collaborating with other users on GitHub. You’ll need to wait until the 2FA enrollment is complete for full access to your account features.

**Comment by ProyectoTuTutorPUJ** [comment]
We (as organization) tried to turn our account as an orgazation, instead of individual one.
We solved by changing the name of my user profile, and then changed it again to the original one.
